frontend/public/components/default-resource.tsx

```diff
@@ -102,22 +104,29 @@ export const DefaultList: React.FC<TableProps & { kinds: string[] }> = (props) =
     ];
   };
 
-  const TableRowForKind: React.FC<RowFunctionArgs<K8sResourceKind>> = ({ obj, customData }) => {
+  const TableRowForKind: React.FC<RowFunctionArgs<TableRowForKindProps>> = ({
+    obj,
+    customData,
+    namespace,
+  }) => {
     const kind = referenceFor(obj) || customData.kind;
     const menuActions = [...Kebab.getExtensionsActionsForKind(kindObj(kind)), ...common];
+    const appliedClusterQuotaReference = referenceForModel(AppliedClusterResourceQuotaModel);
+    const resourceNamespace =
+      kind === appliedClusterQuotaReference ? namespace : obj.metadata.namespace;
```

Comment:
We shouldn't be changing the default resource table. Add a AppliedClusterResourceQuota specific table instead.